### PR TITLE
Use split coordinates for queries

### PR DESCRIPTION
**Pull request by ihnorton**

### CODE REVIEW

File: tiledb/indexing.pyx [hunk 1]
```diff
@@ -138,15 +136,20 @@ cdef dict execute_multi_index(Array array,
...
     cdef uint64_t repeat_count = 0
     cdef uint64_t buffer_bytes_remaining = 0
+    cdef list attrs = list()
+
+    # Coordinate attribute buffers must be set first
+    if return_coord:
+        dims = tuple(array.schema.domain.dim(dim_idx) for dim_idx in \
+                     range(array.schema.ndim))
+        attrs += [QueryAttr(dim.name, dim.dtype) for dim in dims]
+    print("current attrs: ", attrs)
```
Comment (eddelbuettel) [inline tiledb/indexing.pyx:146]: Debug output?

Comment (ihnorton) [reply tiledb/indexing.pyx]: Thanks

**Comment by ihnorton** [comment]
closing in favor of #304

**Comment by ihnorton** [comment]
@normanb I will leave the branch here b/c I think you were building a docker image against it.

**Pull request closed without merging**